scripts/deployment/phase2-assets/collaterals/deploy_morpho_aavev2_plugin.ts

```diff
@@ -0,0 +1,150 @@
+    morphoController: networkConfig[chainId].MORPHO_AAVE_CONTROLLER!,
+    morphoLens: networkConfig[chainId].MORPHO_AAVE_LENS!,
+    rewardsDistributor: networkConfig[chainId].MORPHO_REWARDS_DISTRIBUTOR!,
+    underlyingERC20: networkConfig[chainId].tokens.USDT!,
+    poolToken: networkConfig[chainId].tokens.aUSDT!,
+    rewardToken: networkConfig[chainId].tokens.MORPHO!,
+  })
+
+  const maUSDC = await MorphoTokenisedDepositFactory.deploy({
+    morphoController: networkConfig[chainId].MORPHO_AAVE_CONTROLLER!,
+    morphoLens: networkConfig[chainId].MORPHO_AAVE_LENS!,
+    rewardsDistributor: networkConfig[chainId].MORPHO_REWARDS_DISTRIBUTOR!,
+    underlyingERC20: networkConfig[chainId].tokens.USDC!,
+    poolToken: networkConfig[chainId].tokens.aUSDC!,
+    rewardToken: networkConfig[chainId].tokens.MORPHO!,
+  })
+  const maDAI = await MorphoTokenisedDepositFactory.deploy({
+    morphoController: networkConfig[chainId].MORPHO_AAVE_CONTROLLER!,
+    morphoLens: networkConfig[chainId].MORPHO_AAVE_LENS!,
+    rewardsDistributor: networkConfig[chainId].MORPHO_REWARDS_DISTRIBUTOR!,
+    underlyingERC20: networkConfig[chainId].tokens.DAI!,
+    poolToken: networkConfig[chainId].tokens.aDAI!,
+    rewardToken: networkConfig[chainId].tokens.MORPHO!,
+  })
+
+  await maUSDT.deployed()
+  await maUSDC.deployed()
+  await maDAI.deployed()
+
+  assetCollDeployments.erc20s.maUSDT = maUSDT.address
+  assetCollDeployments.erc20s.maUSDC = maUSDC.address
+  assetCollDeployments.erc20s.maDAI = maDAI.address
+
+  /******** Morpho collateral **************************/
+  const FiatCollateralFactory = await hre.ethers.getContractFactory(
+    "MorphoFiatCollateral"
+  )
+
+  {
+    const collateral = await FiatCollateralFactory.connect(deployer).deploy({
+      priceTimeout: priceTimeout.toString(),
+      chainlinkFeed: networkConfig[chainId].chainlinkFeeds.USDT!,
+      oracleError: fp('0.005').toString(), // 0.5%
+      erc20: maUSDT.address,
+      maxTradeVolume: fp('1e6').toString(), // $1m,
+      oracleTimeout: oracleTimeout(chainId, '3600').toString(), // 1 hr
+      targetName: ethers.utils.formatBytes32String("USD"),
+      defaultThreshold: fp('0.03').toString(), // 3%
+      delayUntilDefault: bn('86400').toString(), // 24h
+    },
+      revenueHiding
+    );
+    assetCollDeployments.collateral.maUSDT = collateral.address
+    deployedCollateral.push(collateral.address.toString())
+
+  }
+  {
+
+    const collateral = await FiatCollateralFactory.connect(deployer).deploy({
+      priceTimeout: priceTimeout.toString(),
+      chainlinkFeed: networkConfig[chainId].chainlinkFeeds.USDC!,
+      oracleError: fp('0.005').toString(), // 0.5%
+      erc20: maUSDC.address,
+      maxTradeVolume: fp('1e6').toString(), // $1m,
+      oracleTimeout: oracleTimeout(chainId, '3600').toString(), // 1 hr
+      targetName: ethers.utils.formatBytes32String("USD"),
+      defaultThreshold: fp('0.03').toString(), // 3%
```

Comment:
the rule we've been using for `defaultThreshold` is oracleError plus 1%, so 1.25% in this case